src/serializer/visitors.js

```diff
@@ -72,7 +72,7 @@ export let ClosureRefReplacer = {
     state.requireStatistics.count++;
     if (state.modified[path.node.callee.name]) return;
 
-    let moduleId = path.node.arguments[0].value;
+    let moduleId = "" + path.node.arguments[0].value;
```

Comment:
I see. Points to that requireReturns should probably be backed by an object, not a map. This all should get reviewed...